lib/Command/Bot/Remove.php

```diff
@@ -43,9 +51,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		$botId = (int) $input->getArgument('bot-id');
 		$tokens = $input->getArgument('token');
 
-		$this->botConversationMapper->deleteByBotIdAndTokens($botId, $tokens);
+		try {
+			$botServer = $this->botServerMapper->findById($botId);
+		} catch (DoesNotExistException) {
+			$output->writeln('<error>Bot could not be found by id: ' . $botId . '</error>');
+			return 1;
+		}
 
+		$this->botConversationMapper->deleteByBotIdAndTokens($botId, $tokens);
 		$output->writeln('<info>Remove bot from given conversations</info>');
+
+		foreach ($tokens as $token) {
+			$event = new BotDisabledEvent($this->roomManager->getRoomByToken($token), $botServer);
```

Comment:
And similarly we need to check if the room existed here:
![grafik](https://github.com/nextcloud/spreed/assets/213943/efb5f2d9-9bb8-4e30-bb65-e664b1aace20)


```suggestion
			try {
				$room = $this->roomManager->getRoomByToken($token);
			} catch (RoomNotFoundException) {
				continue;
			}
			$event = new BotDisabledEvent($room, $botServer);
```

(+ importing the `OCA\Talk\Exceptions\RoomNotFoundException` exception)